README.md

```diff
@@ -87,3 +87,11 @@ The primary supported way of running PeopleMath in a staging or production confi
 Authentication using Firebase is in development.
 Currently, only the frontend is implemented. As long as
```

Comment:
This line is no longer true as of this PR. :) Can you re-check this whole section?

Reply:
I've edited now, not sure about the flag, did we say we'd rather have this as a setting? I skimmed the settings implementation and couldn't find an easy way to implement that.

Reply:
For now I think it is OK for it to be a flag. As and when I roll this out to our internal instance, I may move it to the `Settings`. The way to implement it would be to load the settings from the storage, after the `StorageService` has been instantiated but before the `Server` has.